openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1119,6 +1121,96 @@ Cache-Control: no-store
 }
 ```
 
+# Callback Endpoint {#callback_endpoint}
+
+This endpoint is used by the Wallet to notify the Credential Issuer whether a Credential has been successfully received or not. The Credential Issuer needs to return `callback_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL. The wallet MUST call this endpoint if the Credential Issuer supports it and provides a `callback_id`.
```

Comment:
```suggestion
This endpoint is used by the Wallet to notify the Credential Issuer whether a Credential has been successfully received or not. The Credential Issuer needs to return `callback_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL, but it MUST be supported if `callback_id` is returned in the Credential Response. The wallet MUST call this endpoint if the Credential Issuer supports it and provides a `callback_id`.
```

Reply:
I think `Support for this endpoint is OPTIONAL` is sufficient and it is consistent with the descriptions of other endpoints. if we want to improve this, it should be done in another PR for other endpoints too